eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -168,6 +168,57 @@
           "new": "class com.azure.ai.formrecognizer.documentanalysis.models.(DocumentField|DocumentLine)",
           "justification": "Skip customized getters on class when serialization and deserialization."
         },
+        {
+          "regex": true,
+          "code": "java.annotation.added",
+          "new": "class com.azure.ai.formrecognizer.documentanalysis.models.(DocumentTableCellKind|DocumentSignatureType)",
+          "justification": "Using generated enum class now adds this annotation."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.visibilityReduced",
+          "old": "method void com.azure.ai.formrecognizer.documentanalysis.models.AddressValue::<init>()",
+          "new": "method void com.azure.ai.formrecognizer.documentanalysis.models.AddressValue::<init>()",
+          "oldVisibility": "public",
+          "newVisibility": "private",
+          "justification": "Default constructor made private to prevent direct instantiation."
+        },
```

Comment:
These suppressions shouldn't be done. These are breaking changes and the constructors should be made public.